README.md

```diff
@@ -46,16 +46,13 @@ You can find code samples and tutorials each time that you connect a supported d
 This extension provides several commands in the Command Palette (<kbd>F1</kbd> or <kbd>Ctrl</kbd> + <kbd>Shift</kbd> + <kbd>P</kbd> *or* <kbd>Cmd</kbd> + <kbd>Shift</kbd> + <kbd>P</kbd>) for working with `*.ino` files:
 
 - **Arduino: Board Manager**: Manage packages for boards. You can add 3rd party Arduino board by configuring `Additional Board Manager URLs` in the board manager.
-- **Arduino: Change Baud Rate**: Change the baud rate of the selected serial port.
 - **Arduino: Change Board Type**: Change board type or platform.
-- **Arduino: Change Timestamp Format**: Change format of timestamp printed before each line of Serial Monitor output.
```

Comment:
It looks like this command is still in the product. Are you planning to fully remove it in a future release and make it undocumented in this release?

If the command is on a deprecation path, perhaps we should add something like "[deprecated]" to the command title.

Reply:
No, I don't see a big reason to change it now that it points to the serial monitor setting, I had forgotten to put it back, will fix.